ui/src/components/system/Dashboard.vue

```diff
@@ -907,9 +954,21 @@ export default {
               available_bytes: success.status.memory.SwapTotal
             }
           };
+          
+          context.system.disk = {
+            root: {
+              used_bytes: success.status.disk.root.used,
+              available_bytes: success.status.disk.root.total
```

Comment:
I think this line has to be changed to `success.status.disk.root.available` not `success.status.disk.root.total`

Reply:
Yes, right. It's a typo error. I fix it.

Reply:
@edospadoni I think there is the same problem with ram and swap. Chart is not correct because available_bytes is total size. I commit changes only for disk, tell me if I have to change behaivour also on ram and swap.

Reply:
I don't think so :thinking:

The SWAP is correct:
```js
used_bytes:
    success.status.memory.SwapTotal -
    success.status.memory.SwapFree,
available_bytes: success.status.memory.SwapTotal
```

The RAM is correct:
```js
used_bytes:
    success.status.memory.MemTotal -
    success.status.memory.MemAvailable,
available_bytes: success.status.memory.MemTotal
```

Try to check the values with the command `free -h`

Reply:
![image](https://user-images.githubusercontent.com/17910311/71009584-c81cfb00-20ea-11ea-94e7-bfe0d4570b15.png)

If 2.01 GB are used of 3.7 total GB, the chart percentage should be over 50%. The error is that we use available_bytes as total in the chart column.

Reply:
If 2.01 GB are used of 3.7 total GB, the chart percentage should be over 50%. The error is that we use available_bytes as total in the chart column.

![image](https://user-images.githubusercontent.com/17910311/71009864-3cf03500-20eb-11ea-8418-5d2919c5d233.png)

Reply:
ok, only in the RAM card, SWAP seems right. You can fix also the RAM chart to match the correct value. thanks

Reply:
Can I commit here or do you prefer another PR?

Reply:
Another PR, this PR is focused on disk card, this RAM fix is different, but you can link the same Issue.

Reply:
Opened PR here: https://github.com/NethServer/nethserver-cockpit/pull/197